sale_order_archive/README.rst

```diff
@@ -0,0 +1,98 @@
+==================
+Sale Order Archive
+==================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-workflow/tree/11.0/sale_order_archive
+    :alt: OCA/sale-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-11-0/sale-workflow-11-0-sale_order_archive
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+On a system with a high volume of sales, the number of sale orders displayed in the list view can become huge.
+This module allows to archive Sale Orders that are in status Locked or Cancelled.
+
+If a sale order is archived, it will be hidden from the sale orders list view.
+
+This module only depends on module sale, but it could be used in combination with OCA module 'record_archiver' in order
+to automatically archive old sale orders.
```

Comment:
Could you add a link to this? Am only seeing that module for v7.0 - or is this a roadmap item?

Reply:
Is not yet ported to versions higher that 7.0 AFAIK. Not sure that the link to a V7.0 would be an help.
This paragraph was meant to just explain the potentials of this module.